src/sentry/quotas/redis.py

```diff
@@ -95,16 +101,20 @@ def get_next_period_start(interval):
         client = self.cluster.get_local_client_for_key(six.text_type(project.organization.pk))
         rejections = is_rate_limited(client, keys, args)
         if any(rejections):
+            enforce = True
             worst_case = (0, None)
             for quota, rejected in zip(quotas, rejections):
                 if not rejected:
                     continue
+                if not quota.enforce:
```

Comment:
If both Quota A and Quota B are rejected but only Quota B is *not* enforced, doesn't this mean that the rate limit check will return *not* rate limited even though Quota A *should* be enforced? Is this intentional?